src/js/components/conversation.jsx

```diff
@@ -94,21 +95,24 @@ export class ConversationComponent extends Component {
     };
 
     scrollToBottom = () => {
-        const {shouldScrollToBottom} = this.props;
-        if(!this._isScrolling && (shouldScrollToBottom || this._forceScrollToBottom)) {
+        const {shouldScrollToBottom, quickReplies} = this.props;
+        if (!this._isScrolling && (shouldScrollToBottom || this._forceScrollToBottom)) {
             this._isScrolling = true;
-            const timeout = setTimeout(() => {
```

Comment:
`this` inside the `setTimeout` callback was the wrong one. `this.props.quickReplies` was full when an appUser message would come in when it should be empty.

I tested it and it's working alright without it. I'd say, no `setTimeout` going forward.